test/rulesets/Base/nondiff.jl

```diff
@@ -0,0 +1,7 @@
+@testset "Base/nondiff.jl" begin
```

Comment:
```suggestion
@testset "nondiff.jl" begin
```
testset nesting handles this for you.